lib/main.js

```diff
@@ -275,6 +275,14 @@ const Hydrogen = {
     store.subscriptions.add(disposable);
     return disposable;
   },
+
+  consumeDatatip(datatipService: atom$DatatipService) {
+    const disposable = datatipService.addProvider(
```

Comment:
You are consuming, but also providing. Never thought that this would happen 😆. I think you made the right choice to put it as consumed since the package.json is consuming.